package.json

```diff
@@ -24,8 +24,10 @@
     "Dao Lam <daolam112@gmail.com>",
     "Darren Matsumoto <aeneascorrupt@gmail.com>",
     "Jeremiah Wuenschel <jeremiah.wuenschel@gmail.com>",
+    "Jerry Zhang <thejerryzhang@gmail.com>",
```

Comment:
Thanks for pointing me to where this is configured! https://github.com/screwdriver-cd/generator-screwdriver/pull/32